data-products-dp0-3/table-access-and-queries.rst

```diff
@@ -117,6 +118,38 @@ The N:1 nature of these joins means that the data retrieved can contain columns
 be larger than expected, and take a long time to execute.
 
 
+.. _DP0-3-Table-Access-ADQL-passing-list:
+
+Advice for passing an existing list to a new query
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+LSST Query Services (Qserv) do not support subqueries. Thus, using subqueries is **not recommended** 
+although DP0.3 is not hosted on Qserv. Instead, when having a list of objects in hand either from a 
+previous query or a user-provided catalog, the list, formatted as a python tuple, can be passed to a 
+new query for table joins. The example query below is to retrieve information about individual observations 
+from the ``DiaSource`` and ``SSSource`` tables for indivdual unique objects selected from the ``SSObject`` 
+table and stored in a ``df_uniqueObj`` table from a previous query.
+
+.. code-block:: python
+
+    from lsst.rsp import get_tap_service, retrieve_query
+    service = get_tap_service()
+
+    sId_list = [-9222537907249304995, -9222483995821535577, -9221971933016733299]
+
+    query = """SELECT dia.ssObjectId, dia.diaSourceId, dia.mag,
+    dia.magErr, dia.band, dia.midPointMjdTai,
+    sss.phaseAngle, sss.topocentricDist, sss.heliocentricDist
+    FROM dp03_catalogs_10yr.DiaSource as dia
+    INNER JOIN dp03_catalogs_10yr.SSSource as sss
+    ON dia.diaSourceId = sss.diaSourceId
+    WHERE dia.ssObjectId
+    IN {}
+    """.format(tuple(sId_list))
+
+    results = service.search(query).to_table()
+
```

Comment:
Perhaps we could add a final statement about how this returns a results table with 1915 rows, with 597, 572, and 746 rows for each of the SSObjects, respectively.